### ISSUE TITLE
Retrieving invalidation message in redis-py when client tracking is used

### ISSUE THREAD

**Issue by theyueli** [redis/redis-py#3087]
hello, 

I posted this question on Redis' discussion panel:
https://github.com/redis/redis/discussions/12880

and got no response.. trying to post the same question here as it is more relevant...

I've been trying to play with client tracking in redis-py, I'm using Redis-py 5.0.1 with a local build of the redis main branch. 

I have the following testing code:

```
import redis
from redis import Redis

def our_func(message):
    print(message)

def test_client_tracking_resp_sync():
    client = redis.Redis(host='localhost', port=6379, protocol=3)
    client.ping()
    resp = client.client_tracking_on()
    resp = client.execute_command("GET FOO", push_handler_func=our_func)
    resp = client.execute_command("SET FOO 100", push_handler_func=our_func)
    print(resp)
    resp = client.execute_command("GET FOO", push_handler_func=our_func)
    print(resp)
    resp = client.execute_command("FLUSHDB")
    print(resp)

test_client_tracking_resp_sync()
```

I'm trying to follow the suggestion here:
https://redis-py.readthedocs.io/en/stable/resp3_features.html

But it seems I can't retrieve the invalidation message as the print statement always just prints out the command result only:

```
b 'OK'
b '100'
True
```

Did I miss anything here? 

The same flow works well with correctly printed invalidation message on redis-cli.

**Comment by dvora-h** [comment]
Hi @theyueli,
As of now I don't think it's possible.
but as you can see in #3038 and #3089 this is something we are working on now - to add client side caching to redis-py so when this work will be done you will can use this feature

**Comment by theyueli** [comment]
thanks a lot for clarifying! please keep me posted and i will definitely be willing to test it when it is ready.

**Comment by dvora-h** [comment]
@theyueli We released a beta versions (5.1.0b2 already released and 5.1.0b3 will come in the next 2 days) with client side caching support, your more than welcome to test it and you can feel completely free to bother me with any questions you have about it.

**Comment by theyueli** [comment]
> @theyueli We released a beta versions ([5.1.0b2](https://github.com/redis/redis-py/releases/tag/v5.1.0b2) already released and 5.1.0b3 will come in the next 2 days) with client side caching support, your more than welcome to test it and you can feel completely free to bother me with any questions you have about it.

thank you @dvora-h ! I will give it a try soon, will keep you posted.
